dask/dataframe/groupby.py

```diff
@@ -451,6 +463,13 @@ def _finalize_std(df, count_column, sum_column, sum2_column, ddof=1):
     return np.sqrt(result)
 
 
+def _determine_levels(index):
+    if isinstance(index, (tuple, list)) and len(index) > 1:
+        return list(range(len(index)))
+    else:
+        return 0
```

Comment:
It would be nice to have some text, perhaps just as a long comment in the code, about how these small functions `_determine_levels`, `_get_groupby_chunk_args`, etc. work together.  This would help when maintaining this code and would also help newer developers.